Topics/Development_Process.md

```diff
@@ -4,6 +4,8 @@
 ### [Automated Frontend Deployment with Vercel](./Development_Process/Frontend_Automated_Deployment_Vercel.md)
 ### [Automated Testing](./Development_Process/Automated_Testing.md)
```

Comment:
There seems to already be a section on Automated Testing here. Do you think QA testing is different enough to be it's own section or should we combine testing? Here's the PR of automated testing for reference: https://github.com/learning-software-engineering/learning-software-engineering.github.io/pull/31/files

Reply:
I was going to include automated testing vs manual testing in my article until I saw this. I think we cover different topics but it does fall into a larger category of testing.
I think combining it might be a good idea.

Reply:
Let me know what you decide and I'll re-review your changes (if you make any). I would also make note in your PR of your decision and why. Thanks!

Reply:
I made the changes according to the recommendation from piazza.
https://piazza.com/class/lc6otyey9pw78t/post/98

Reply:
That's a good idea, thanks for being diligent and asking on Piazza 😄